Dockerfile

```diff
@@ -1,44 +1,15 @@
-# ----------------------------------------------------------
-#       Dockerfile to build base working Ubuntu image
-# ----------------------------------------------------------
+# -----------------------------------------------------------------------------
+#       Dockerfile to build Kaldi gstreamer server with tue-env
+# -----------------------------------------------------------------------------
 
-# Set the base image to Ubuntu 16.04
-FROM ubuntu:16.04
-
-# File Author / Maintainer
-MAINTAINER Arpit Aggarwal
-
-# Make image to be noninteractive to use with CI
-# Comment this line when not using this image with CI
-ENV DEBIAN_FRONTEND=noninteractive
-
-# Set environment variable USER
-ENV USER=hero
+# Set the base image to Ubuntu with tue-env
+FROM tueroboticsamigo/tue-env:cleanup_docker-and-ci
```

Comment:
master??

Reply:
After that Docker PR is merged.

Reply:
Please add a comment, so we don't forget this.